index.mako

```diff
@@ -1572,6 +1572,10 @@ which may be faster, use something like
 
 [XDG Base Directory Specification]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
 
+If your CI environment uses immutable caches, the pre-commit repository should
+be versioned using both the contents of your configuration file (typically
+`.pre-commit-config.yaml`) and the output of the command `python -VV`.
```

Comment:
this isn't always completely true, but is a good estimate for the `healthy()` check

Reply:
Good info. Added a note below with that caveat.